paper-card.html

```diff
@@ -58,7 +58,7 @@
 
 <dom-module id="paper-card">
   <template>
-    <style>
+    <style include="paper-material">
```

Comment:
I have no opinion. I just copied what Steve did in `paper-button`, which was this. I think he didn't change `paper-material` at all, so the polymer example wouldn't work without first changing `paper-material`. Ughhhhhh